packages/smooth_app/lib/pages/onboarding/consent_analytics_page.dart

```diff
@@ -182,4 +109,61 @@ class ConsentAnalytics extends StatelessWidget {
       OnboardingFlowNavigator.getNextPage(OnboardingPage.CONSENT_PAGE),
     );
   }
+
+  Widget _buildButton(
+    BuildContext context,
+    Color btnColor,
+    String label,
+    IconData icon,
+    bool isAccepted,
+  ) {
+    final Size size = MediaQuery.of(context).size;
+    final LocalDatabase localDatabase = context.watch<LocalDatabase>();
+    final UserPreferences userPreferences = context.watch<UserPreferences>();
+    return InkWell(
+      onTap: () {
+        _analyticsLogic(
+          isAccepted,
+          userPreferences,
+          localDatabase,
+          context,
+        );
+      },
+      child: Padding(
+        padding: EdgeInsets.fromLTRB(
```

Comment:
`EdgeInsets.symmetric(horizontal: 2.0)` would be easier to read.

Reply:
Sure @monsieurtanuki, I'll keep this in mind for upcoming PRs.